specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-03-01-preview/subscription.json

```diff
@@ -0,0 +1,312 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-03-01-preview",
+    "title": "SubscriptionManagementClient",
+    "description": "Subscription management client provides an interface to create and manage Azure subscriptions programmatically."
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Subscription/operations": {
```

Comment:
Shouldn't this swagger contain all of the things from the stable subscriptions.json swagger as well?